tdei-ui/package.json

```diff
@@ -20,11 +25,14 @@
     "react-bootstrap": "^2.7.0",
     "react-copy-to-clipboard": "^5.1.0",
     "react-dom": "^18.2.0",
+    "react-dropzone": "^14.2.3",
     "react-intersection-observer": "^9.4.1",
     "react-query": "^3.39.2",
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.6.1",
     "react-scripts": "5.0.1",
+    "react-select": "^5.8.0",
+    "react-toastify": "^10.0.5",
```

Comment:
since we are using bootstrap and now added MUI , dont they have select and toast components ?

Reply:
They do, but I have seen a requirement where toast has to dismiss after 30 seconds. This toast by default provides it. Also, easily customizable.

Reply:
Resolved and replaced with bootstrap toast. 
<img width="1470" alt="Screenshot 2024-03-28 at 6 10 31 PM" src="https://github.com/TaskarCenterAtUW/TDEI-user-management-front-end/assets/142565248/935bb38f-d411-46e8-b07a-e8d0f0b94daa">

<img width="1470" alt="Screenshot 2024-03-28 at 6 24 58 PM" src="https://github.com/TaskarCenterAtUW/TDEI-user-management-front-end/assets/142565248/05b9dcf0-0cf9-4c54-ab03-77d2fa4aaafa">